### ISSUE TITLE
Uploading large files

### ISSUE THREAD

**Issue by Kakakirbz** [richardgirges/express-fileupload#93]
Hello! I am in need of big help. I am working on a module of my thesis that specifically can upload multiple files in one input. I tried uploading videos. I can upload videos that has a size of 20mb but when I tried and upload more than 100mb, it returns an error message. The problem is, I can't seem to upload files that are large.

Here is the error message
![image](https://user-images.githubusercontent.com/22406521/45582436-75023a00-b8e2-11e8-9226-fcee2b22f231.png)

I hope this issue could be answered ASAP. Thank you ahead!

**Comment by randomdipesh** [comment]
exactly same issue here 
![screenshot from 2018-09-19 08-22-24](https://user-images.githubusercontent.com/25702719/45727914-6fa83680-bbe5-11e8-878e-3df817e0cc8d.png)

**Comment by randomdipesh** [comment]
Here is how I fixed it :+1: 
node_modules>express-fileupload>lib>index.js  and removed md5= md5(buf) on line 146
Its working now

**Comment by senakafdo** [comment]
A suggestion for the devs would be to replace `md5: md5(buf),` with `md5: function() { return md5(buf); },` such that somebody can then use it as: `req.files.foo.md5()` - it not only solves this OOM issue but also makes the execution faster.

Also, a slightly better tweak compared to what @randomdipesh suggested would be to include the following in the code:
```
const mock = require('mock-require');
mock('md5', 'empty/functionThatReturnsArgument');
const upload = mock.reRequire('express-fileupload');
```
And add the following as dependencies:
```
    "mock-require": "3.0.2",
    "empty": "0.10.1",
```

**Comment by richardgirges** [comment]
I'm going to take @senakafdo's advice and update md5 to become a method for execution at will, as opposed to a property that's always available on the file object. 

I'll try to get this pushed up in a major release sometime in the next week

**Comment by richardgirges** [comment]
Should be fixed in latest release: `v1.0.0-alpha.1`
